File: python/my_queue.py
from queue import Queue

queue = Queue()

queue.put(1)
queue.put(2)
queue.put(3)

print("Size of stack is %d." % queue.qsize())
print("Leftmost element of the stack is %d." % queue.get())

queue.put(4)
queue.put(5)

print("FIFO:", end="")
while not queue.empty():
    print(queue.get(), end=" ")

File: cpp/my_queue.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){

    queue<int> q;
    q.push(1);
    q.push(2);
    q.push(3);
    cout<<q.front()<<endl;
    q.pop();
    cout<<q.front()<<endl;
    q.push(4);
    q.push(5);
    cout<<q.empty()<<endl;

    return 0;
}
